debbuild

```diff
@@ -238,6 +265,34 @@ sub load_static_config {
 }
 
 
+sub read_multiline {
```

Comment:
actually not just macro, but also reading spec file, but only in case of reading multiline macro defined in the spec, so it makes sense to rename it :+1: